### PR TITLE
Drupal 8 with theme

### PR DESCRIPTION
**Pull request by LightGuard**
Using Drupal 8 now, theme PoC in place.

### CODE REVIEW

**Comment by ollyjshaw** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
Needs rebasing.

**Comment by paulrobinson** [comment]
@LightGuard after rebase, is this ready for review and then merge?

**Comment by LightGuard** [comment]
No, it needs to be replaced.

On Tuesday, November 24, 2015, Paul Robinson <notifications@github.com>
wrote:

> @LightGuard <https://github.com/LightGuard> after rebase, is this ready
> for review and then merge?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/562#issuecomment-159258654>
> .
>


-- 
Sent from Gmail Mobile

**Comment by LightGuard** [comment]
Replaced with #562

**Comment by paulrobinson** [comment]
@LightGuard what's the status of this PR? Ready for review?

**Comment by LightGuard** [comment]
Yeah. On-going, but yeah, you can review it

On Tuesday, December 1, 2015, Paul Robinson <notifications@github.com>
wrote:

> @LightGuard <https://github.com/LightGuard> what's the status of this PR?
> Ready for review?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/562#issuecomment-160984933>
> .
>


-- 
Sent from Gmail Mobile

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by paulrobinson** [comment]
@LightGuard are you happy for this to be merged, after a review?

**Comment by paulrobinson** [comment]
@ollyjshaw please can you review?

File: _docker/lib/options.rb [hunk 1]
```diff
@@ -55,7 +55,7 @@ def self.parse(args)
 
       opts.on('-u', '--drupal', 'Start up and enable drupal') do |u|
         tasks[:drupal] = true
-        tasks[:supporting_services] += %w(drupal drupalmysql)
+        tasks[:supporting_services] += %w(drupal drupalpgsql)
```
Comment (ollyjshaw) [inline _docker/lib/options.rb]: I think you should update test_options.rb as well

**Comment by ollyjshaw** [comment]
Minor comment.  I assume the binary is ok. 
Overall :+1:

**Pull request merged**